Source/Public/LevelStatsCollector.h

```diff
@@ -1,9 +1,54 @@
 ﻿#pragma once
 
+#include <ChartCreation.h>
 #include <CoreMinimal.h>
 
 #include "LevelStatsCollector.generated.h"
 
+class MAPMETRICSGENERATION_API FCustomPerformanceChart final : public FPerformanceTrackingChart
+{
+public:
+    FCustomPerformanceChart( const FDateTime & in_start_time, const FString & in_chart_label, const FString & in_output_path ) :
+        FPerformanceTrackingChart( in_start_time, in_chart_label ),
+        CustomOutputPath( in_output_path )
+    {
+    }
+
+    void DumpFPSChartToCustomLocation( const FString & in_map_name )
+    {
+        TArray< const FPerformanceTrackingChart * > charts;
```

Comment:
Why not put the definition in the cpp?

Reply:
My bad, old habits for small helper classes like that. I will put it in the cpp.

Reply:
Done.